beanstalk_worker/constants.py

```diff
@@ -18,5 +18,9 @@
 
 LINTER_RESULTFILE = "linter_results.txt"
 
-LOGS_URL_BASE= "https://registry.centos.org/pipeline-logs/"
+LINTER_STATUS_FILE = "linter_status.json"
```

Comment:
No. This file is going to have status of execution of linter. For example, if it has failed or passed. If passed, the json file will contain the path of result file. This file actually is `job_info` data with some additional entries. This was needed to by (last) mail server module to understand, if earlier modules executed properly, and their final state.